Sources/Networking/HTTPClient/HTTPClient.swift

```diff
@@ -61,7 +61,7 @@ class HTTPClient {
         completionHandler: Completion<Value>?
     ) {
         #if DEBUG
-        guard !self.systemInfo.dangerousSettings.internalSettings.forceServerErrors else {
+        guard !self.systemInfo.forceServerErrors else {
```

Comment:
`InternalSettings` aren't mutable, but stubbing this in `MockSystemInfo` will allow us to "change" the flag at runtime in tests.